code/src/patches.s

```diff
@@ -1125,6 +1125,22 @@ MidoCheckDekuTreeClearFive_patch:
 CorrectCompassChests_patch:
     bl hook_CheckCurrentDungeonMode
 
+.section .patch_FishingNewRecord
+.global FishingNewRecord_patch
+FishingNewRecord_patch:
+    mov r0,#0x0
+    cmp r0,#0x1
+
+.section .patch_FishingSizeIgnoreChild
+.global FishingSizeIgnoreChild_patch
+FishingSizeIgnoreChild_patch:
+    nop
+
+.section .patch_FishingSizeIgnoreAdult
+.global FishingSizeIgnoreAdult_patch
+FishingSizeIgnoreAdult_patch:
+    nop
+
```

Comment:
im pretty meh on making it so any fish works. its fine but i think we should revisit this in the future

Reply:
Yeah, it isn't great, but I think it's better to do this now and revisit it later at some point.